app/assets/stylesheets/application.css

```diff
@@ -1,4 +1,5 @@
 @import "bootstrap";
```

Comment:
Funktioniert das richtig, in einem CSS-File? Müsste das nicht SCSS Sein?

Reply:
Ja ich weiß auch nicht warum das so ist, Ich habe ewig probiert mit scss.
Aber so funktioniert es jetzt.

Reply:
OK, habe es gefunden. Du hast die falsche Version von Bootstrap hinzugefügt und entsprechend falsch konfiguriert. 
https://github.com/WilliBoelke/BelegarbeitWebentwicklung/blob/master/Gemfile#L35.